_notebooks/2020-12-28-CoreferenceResolution.ipynb

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
<p>![](https://i.imgur.com/fnexWCa.png)</p><p> </p><p>Maybe we can improve the error message to not hard code specific function names?</p><p> </p><pre class="ql-syntax" spellcheck="false">There is no clean_text for the chapter.                     Please run book.clean_raw_text or manually add                    custom cleaned text by iterating through chapters
</pre><p> </p><p> </p><p>Here is an alternative, which you can add to the Exception you raise. </p><pre class="ql-syntax" spellcheck="false">The recommended approach to restore sentence sentence boundaries before coreference resolution. You may save the  in &lt;var-name&gt; of each book. 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/projectnoor/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>